visualizations/lorentz-transformation/static/style.css

```
@@ -0,0 +1,357 @@
+  --border: var(--atlas-viz-border);
+  --accent: var(--atlas-viz-accent);
+  --ink: #292c30;
+  --muted: #687177;
+  --faint: #92979b;
+  --frame-base: #586a76;
+  --frame-prime: #168a91;
+  --event: #c94f49;
+  --light: #d09a3e;
+  --proper-interval: #bf7042;
+  --result: #8b4b3f;
+  --serif: "Atlas Source Serif 4", "Iowan Old Style", Georgia, serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)